README.md

```diff
@@ -2,15 +2,22 @@ etcd-defrag
 ======
 ## Table of Contents
 
-- **[Overview](#overview)**
-- **[Examples](#examples)**
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Overview](#overview)
+- [Examples](#examples)
   - [Example 1: run defragmentation on one endpoint](#example-1-run-defragmentation-on-one-endpoint)
   - [Example 2: run defragmentation on multiple endpoints](#example-2-run-defragmentation-on-multiple-endpoints)
   - [Example 3: run defragmentation on all members in the cluster](#example-3-run-defragmentation-on-all-members-in-the-cluster)
-- **[Defragmentation rule](#defragmentation-rule)**
-- **[Container image](#container-image)**
-- **[Contributing](#contributing)**
-- **[Note](#note)**
+  - [Example 4: Kubernetes CronJob](#example-4-kubernetes-cronjob)
```

Comment:
I think it deserves a top level section, e.g. "Integration with Kubernetes"

Reply:
good point, will do.